website/content/en/pre-docs/deploy-guide/_index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Deployment Guide"
+linkTitle: "Deployment Guide"
+weight: 50
+---
+
+Introducing new software the modifies how Kubernetes operates is complex, and no single solution exists. 
+
+This guide describes how Karpenter interacts with external resources, and modifies cluster behavior. 
+
+## Resources Dependencies
+
+### Cluster on AWS
+
+Currently, Karpenter creates nodes (EC2 Instances) through the AWS cloud provider module, the only such module implemented. Clusters on other cloud providers are contemplated, but are not currently supported. 
+
+Theoretically, a control plane could be hosted separately from Karpenter provisioned AWS instances. However, this complicates identity and authorization substantially. The Karpenter controller (i.e., agent) would need authorization with AWS APIs to provision new instances, and permissions with the control plane to add the new nodes. 
+
+Additionally, the intricacies of securing control plane communication between clouds presents another significant problem. 
+
+The getting started guide covers [configuring EKS to accept newly provisioned instances as nodes](../getting-started/#create-the-karpenternode-iam-role) (see, *KarpenterNodeRole* and *system:bootstrappers*). If using a self-managed (or alternatively provided) control plane, this must be manually handled. Consult the [*User Data - Autoconfigure* section](../cloud-providers/aws/launch-templates/#user-data---autoconfigure) of *Launch Templates* for more information on joining nodes to a cluster.

```

Comment:
In the second sentence what does "alternatively provided" mean? Isn't that the same as self-managed?

Reply:
is there such a thing as a 3rd party managed control plane on AWS? ROSA perhaps? Regardless, I just struck "alternatively provided"